### ISSUE TITLE
Support for Master Slave Write and read

### ISSUE THREAD

**Issue by matrixbegins** [balderdashy/sails-mysql#177]
Just curious. Only using adapter I can configure two mysql connections in Application, but will ORM support the read form salve and write to master seamlessly ?

**Comment by awm33** [comment]
I'm also interested in using this adapter in a mysql master-slave cluster. The only thing I can thing of is using two instances of the ORM, one for reads and one for writes. If I have to use more than one read server, I'm not sure how to load balance.

I would be nice if it had built in support for read slaves, like sequelize

**Comment by devinivy** [comment]
For now, there is another option of hacking around with using multiple connections on a collection.  See my post here: https://github.com/balderdashy/waterline-docs/issues/45 .

I would love to see support in this adapter to use one master for writes and multiple slaves for reads.  It wouldn't be terribly hard to implement using https://github.com/felixge/node-mysql#poolcluster .

**Comment by sailsbot** [comment]
Thanks for posting, @matrixbegins.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by icodeforlove** [comment]
Any plan on adding this? Sequelize has supported this for quite some time now.

**Comment by icodeforlove** [comment]
I added a POC branch with replication support (#267)
